segmentation_models_pytorch/encoders/timm_gernet.py

```diff
@@ -0,0 +1,120 @@
+from timm.models import ByobCfg, BlocksCfg, ByobNet
+
+from ._base import EncoderMixin
+import torch.nn as nn
+
+
+class GERNetEncoder(ByobNet, EncoderMixin):
```

Comment:
This inherits from byobnet as repvgg so we should consider adding generic ByoBnetEncoder to comply with future timm changes https://github.com/rwightman/pytorch-image-models/blob/d8e69206be253892b2956341fea09fdebfaae4e3/timm/models/byobnet.py#L645